edge-rtac/src/main/resources/domain/edge-rtac/features/rtac.feature

```diff
@@ -0,0 +1,148 @@
+    * def secondInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def itemStatusName = 'Checked out'
+    * def createSecondItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem') { itemId: 68640328-788e-53fc-9c3c-af39e243f3b7 }
+    * def periodicalStatus = true
+    * def rtacBatchrequest = read('samples/rtac-batch-request-entity.json')
+
+    Given path 'rtac-batch'
+    And request rtacBatchrequest
+    When method POST
+    Then status 200
+    And match response.holdings..id contains [ 68640328-799e-43fc-9c3c-af39e243f3b7 , 68640328-788e-53fc-9c3c-af39e243f3b7 ]
+    And match response.holdings..status contains [ Available , Checked out ]
+
+  Scenario:For periodical/serial, return holdings and item information including availability for each instance UUID included in request WHEN &fullPeriodicals=true
+#   1st instance
+    * def instanceId = call random_uuid
+    * def servicePointId = call random_uuid
+    * def locationId = call random_uuid
+    * def holdingId = call random_uuid
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def firstInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def createFirstItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem') { itemId: 68640368-799e-43fc-9c3c-af39e243f3b7 }
+
+#   2nd instance
+    * def instanceId = call random_uuid
+    * def servicePointId = call random_uuid
+    * def locationId = call random_uuid
+    * def holdingId = call random_uuid
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def secondInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def itemStatusName = 'Checked out'
+    * def createSecondItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem') { itemId: 68640328-788e-53fd-9c3c-af39e243f3c7 }
+    * def periodicalStatus = true
+    * def rtacBatchrequest = read('samples/rtac-batch-request-entity.json')
+
+    Given path 'rtac-batch'
```

Comment:
The scenario name states `WHEN &fullPeriodicals=true`, is that intended to be a parameter to the request to edge-rtac?

Reply:
i guess no, **fullPeriodicals=true** is added in  request payload file. PFB 

  "instanceIds": [
    "#(firstInstanceId)",
    "#(secondInstanceId)"
  ],
  **"fullPeriodicals": "#(periodicalStatus)"**

Reply:
The edge-rtac API includes this [parameter](https://github.com/folio-org/edge-rtac/blob/46e58fcc18be555c5ec84658012d1367eb8d77a0/ramls/edge-rtac.raml#L29)

Have you asked the PO how these scenarios are intended to use the periodical information?